docs/guide/acquiring_data/jsoc.rst

```diff
@@ -197,78 +188,67 @@ If you want to get a manual set of keywords in the response object, you can pass
 :meth:`~sunpy.net.base_client.BaseQueryResponseTable.show` method.
 
     >>> res = Fido.search(a.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'),
-    ...                   a.jsoc.Series('hmi.v_45s'), a.jsoc.Notify('sunpy@sunpy.org'))  # doctest: +REMOTE_DATA
+    ...                   a.jsoc.Series('hmi.v_45s'))  # doctest: +REMOTE_DATA
     >>> res.show('TELESCOP', 'INSTRUME', 'T_OBS')  # doctest: +REMOTE_DATA
-    [<QTable length=81>
+    [<sunpy.net.jsoc.jsoc.JSOCResponse object at ...>
     TELESCOP  INSTRUME           T_OBS
-    str7     str10             str23
     -------- ---------- -----------------------
-    SDO/HMI HMI_FRONT2 2014.01.01_00:00:37_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:01:22_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:02:07_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:02:52_TAI
-        ...        ...                     ...
-    SDO/HMI HMI_FRONT2 2014.01.01_00:57:37_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:58:22_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:59:07_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_00:59:52_TAI
-    SDO/HMI HMI_FRONT2 2014.01.01_01:00:37_TAI]
+     SDO/HMI HMI_FRONT2 2014.01.01_00:00:37_TAI
```

Comment:
Why increase the number of rows in the output here?

Reply:
I just copied the raw output might as well cut it back down :+1: